include/picongpu/plugins/openPMD/Parameters.hpp

```diff
@@ -5,14 +5,21 @@
 
 namespace picongpu::openPMD
 {
+    /*
+     * No default values here since those are registered in the
+     * plugins::multi::Option data members of openPMDWriter::Help.
+     * The openPMD plugin will automatically those default values here.
```

Comment:
Thanks for spotting. Below is the wording that I was going for, there is not much of a choice going on in this logic.
```suggestion
     * The openPMD plugin will automatically use those default values here.
```